google-chart.html

```diff
@@ -52,6 +52,7 @@
 <dom-module id="google-chart">
   <link rel="import" type="css" href="google-chart.css">
   <template>
+    <google-chart-loader id="loader" type="[[type]]"></google-chart-loader>
```

Comment:
would it be worth wrapping this in a template dom-if so only instantiate when `type` is set?